src/215-Kth-Largest-Element-In-An-Array/215._Kth_Largest_Element_In_An_Array.java

```diff
@@ -0,0 +1,26 @@
+package leetcode;
+
+import java.util.Comparator;
+import java.util.PriorityQueue;
+
+public class FindKthLargest_215 {
```

Comment:
Also, It's better to add some document comments here to explain why we use priority queue.